lib/grape/validations/types/custom_type_coercer.rb

```diff
@@ -162,15 +162,15 @@ def enforce_symbolized_keys(type, method)
             lambda do |val|
               method.call(val).tap do |new_value|
                 new_value.each do |item|
-                  Hashie.symbolize_keys!(item) if item.is_a? Hash
```

Comment:
It's updating the values in place it looks like. If removing it doesn't cause any specs to fail I would start by writing a test that hits it, then change the behavior.